src/app/demos/toggle-switch/toggle-switch-demo.component.html

```diff
@@ -0,0 +1,60 @@
+<p>
+  <sky-toggle-switch
+    [checked]="true"
```

Comment:
The first two toggle switches seem weird to me. When I click the first toggle switch to switch it from "on" to "off," the text beside it remains 'Switch is "on" ' Likewise, when I click the second toggle switch to switch it from "off" to "on," the text beside it remains 'Switch is "off" ' So basically, as soon as someone interacts with those toggle switches, the labels are inaccurate. I think it would sense to combine these toggle switch into a single toggle switch with a label that alternates between 'Switch is "on" ' and 'Switch is "off" ' But if that's not feasible, I'd change the existing first two toggle switches to make them non-interactive so that we don't have the incorrect labels as soon as someone selects them.